app/Http/Controllers/AuthorController.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Actions\Project\UpdateProject;
+use App\Models\Author;
+use App\Models\Project;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Http;
+
+class AuthorController extends Controller
+{
+    public function getAuthorsbyDOI(Request $request)
+    {
+        $doi = $request->get('doi');
+        $getDetailsbyDOIApi = config('app.get_details_by_DOI_api');
+        $url = $getDetailsbyDOIApi.'/'.$doi;
+        $response = Http::get($url, [
+        ]);
+        $authorsList = [];
+        $author = null;
+        $response = json_decode($response, true);
+        if ($response && $response['status'] == 'ok') {
+            $authorRes = $response['message']['author'];
+            foreach ($authorRes as $item) {
+                $author['given_name'] = array_key_exists('given', $item) ? $item['given'] : null;
+                $author['family_name'] = array_key_exists('family', $item) ? $item['family'] : null;
+                $author['affiliation'] = array_key_exists('affiliation', $item) ? $item['affiliation'][0]['name'] : null;
+                $author['orcid_id'] = array_key_exists('ORCID', $item) ? $item['ORCID'] : null;
+                array_push($authorsList, $author);
+            }
+
+            return $authorsList;
+        } else {
+            return null;
+        }
```

Comment:
Unless we have a private key-based API request and cross-check with the database entries, it's advisable to move this fetch request to the front end - Run it from the javascript component using Axios.